addon/route.js

```diff
@@ -1,17 +1,21 @@
 import Ember from 'ember';
 
-var route = function(funcs) {
+var route = function(props) {
   return function wrapWithRoute(IncomingRoute) {
     var WrappedRoute = IncomingRoute || Ember.Route;
     return WrappedRoute.extend({
       redux: Ember.inject.service('redux'),
       init() {
         var redux = this.get('redux');
         var route = this;
-        Object.keys(funcs).forEach(function(func) {
-          route[func] = function(...args) {
+        Object.keys(props).forEach(function(key) {
+          if (typeof props[key] !== 'function') {
+            route[key] = props[key]
```

Comment:
also ... why didn't eslint complain during the build? should we alter that as part of this PR to ensure less bike shedding in future PRs?